ape_infura/provider.py

```diff
@@ -34,6 +35,26 @@ def __init__(self):
 
 class Infura(Web3Provider, UpstreamProvider):
     network_uris: dict[tuple[str, str], str] = {}
+    api_keys: set[str] = set()
```

Comment:
For security reasons, we may want to (eventually) make this `__`-prefixed as well.
Since `.uri` contains the key, I am not worried about it though.

Reply:
Done